src/vm/core.zig

```diff
@@ -40,6 +40,10 @@ pub const CairoVM = struct {
     is_run_finished: bool,
     /// VM trace
     trace_context: TraceContext,
+    /// Current Step
```

Comment:
Are these changes in the core file normal? Maybe the branch is not up to date with main?

Reply:
think i accidently mixed this pr with my other pending pr will fix